applications/API/tests/test_forms.py

```
@@ -0,0 +1,288 @@
+
+class InsultReviewFormTest(TestCase):
+    """Test suite for InsultReviewForm validation."""
+
+    def setUp(self):
+        """Set up test fixtures before each test method."""
+        self.insult_category = InsultCategory.objects.create(
+            category_key="TEST", name="Test Category"
+        )
+        self.admin_user = User.objects.create_user(
+            username="admin_testuser",
+            password="testpass123",
```

Ruff (0.12.2)

Possible hardcoded password assigned to argument: "password"

(S106)